### PR TITLE
chore(ci): move to github actions

### PR DESCRIPTION
**Pull request by cristianossd**
References:
- https://github.com/Level/community/issues/99

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [8, 10, 12]
```
Comment (vweevers) [inline .github/workflows/test.yml:8]: Note to self, we'll bump this later as part of https://github.com/Level/community/issues/98.

**Review by vweevers — APPROVED** [review]

**Review by vweevers — CHANGES_REQUESTED** [review]
Can you run `npm rm coveralls --save-dev`? We no longer need it. Thanks!

**Comment by cristianossd** [comment]
I did, @vweevers, the `coveralls` package already is not present in `package.json`

**Comment by vweevers** [comment]
Oh, so coverage wasn't even working here 😄 Great, thanks!

**Review by vweevers — APPROVED** [review]

**Pull request merged**